database/buildSchema/43_views_functions_triggers.sql

```diff
@@ -1,3 +1,5 @@
+-- SCHEMA to hide internal functionality from postgraphile
+CREATE SCHEMA IF NOT EXISTS private;
```

Comment:
Should this be in migrations ?

Reply:
Actually, maybe not. Perhaps even DROP the whole schema first, so we know the policy functions are getting created from scratch each time.

Reply:
this would mean row level policies will need to be rebuilt on startup or after every migration, this happens on startup now and snapshot load, would that be all of the use cases ?

Reply:
https://github.com/msupply-foundation/conforma-server/pull/1110/commits/57765679c5e3bdd2b3fcadfdd4e75e16d5f02257